app/src/main/java/com/infomaniak/mail/ui/newMessage/NewMessageViewModel.kt

```diff
@@ -302,11 +328,7 @@ class NewMessageViewModel @Inject constructor(
             doc.split(MessageBodyUtils.INFOMANIAK_FORWARD_QUOTE_HTML_CLASS_NAME, bodyWithQuote)
         }
 
-        draft.apply {
-            uiBody = body.htmlToText()
```

Comment:
Why no more `.htmlToText()` ?

Reply:
Now that we use a rich html editor the input of the editor is not text anymore but html which is what's used here instead now